compio-runtime/Cargo.toml

```diff
@@ -41,6 +41,7 @@ futures-util = { workspace = true }
 once_cell = { workspace = true }
 slab = { workspace = true, optional = true }
 smallvec = "1.11.1"
+socket2 = { workspace = true }
```

Comment:
Why do we need socket for runtime?

Reply:
`impl TryClone for socket2::Socket`

Reply:
Add a feature and enable it in compio-net

Reply:
Don't worry about that, `compio-driver` has already imported it:)